flake.nix

```diff
@@ -106,6 +106,9 @@
         "github:input-output-hk/Win32-network/3825d3abf75f83f406c1f7161883c438dac7277d";
       flake = false;
     };
+    plutus-config = {
+      url = "git+ssh://git@github.com/mlabs-haskell/plutus-config.git?rev=599266c16d5575e640721c542b256902b2d11aae";
```

Comment:
Do we need the `git+ssh://git@` part? Is plutus-config open source?

Reply:
It is a private repo. We need it for Hercules CI.

Reply:
hmm... i think there might users of this library outside MLabs (for example users of plutip). Could we open source that one too?